src/library_pthread.js

```diff
@@ -242,16 +281,25 @@ var LibraryPThread = {
             Module['print']('Thread ' + e.data.threadId + ': ' + e.data.text);
           } else if (e.data.cmd === 'printErr') {
             Module['printErr']('Thread ' + e.data.threadId + ': ' + e.data.text);
-          } else if (e.data.cmd == 'alert') {
+          } else if (e.data.cmd === 'alert') {
             alert('Thread ' + e.data.threadId + ': ' + e.data.text);
           } else if (e.data.cmd === 'exit') {
-            // todo 
+            // Grab back ownership of all OffscreenCanvases that the child worker had in it.
```

Comment:
This code looks like a left-over, since `threadExit` function currently does not pass offscreen canvases in the 'exit' message, but uses explicit 'objectTransfer' message instead.